zeno/server.py

```diff
@@ -198,6 +199,12 @@ def embed_exists(model: str):
     def project_embed_into_2D(req: EmbedProject2DRequest):
         return project_into_2D(zeno.df, zeno.id_column, req.model, req.column)
 
+    @api_app.post(
+        "/slice-finder-project", tags=["zeno"], response_model=str
+    )  # response_model=List[SliceMetric]
+    def project_find_available_slices(req: SliceFinderRequest):
+        return zeno.slice_finder(req)  # TODO: ADD SOMETHING HERE
```

Comment:
Related to above, change this to a non-zeno call to a function in the processing folder.